index.ts

```diff
@@ -71,7 +72,27 @@ const SeverityAnnotationLevelMap = new Map<RuleSeverity, "warning" | "failure">(
     } else {
       const linter = new Linter(options);
 
-      const files = glob.sync(pattern!);
+      let files = glob.sync(pattern!);
```

Comment:
It would be nice if this block can be cleaned.

e.g.
```typescript
const files = await (async () => {
  const matched = glob.sync(pattern!);
  if (!onlyChanged) { 
    return matched;
  }
  
  const changedFiles = ctx.action === "pull_request" ? 
    (await octokit.pulls.listFiles(options)).data :
    await getChangedFilesSomeHow();

  return matched.filter((f) => changedFiles.includes(f));
})();
```

Reply:
Why is this better? We are already in async function.